src/modules/launcher/Wox.Plugin/ToolTipData.cs

```diff
@@ -16,7 +16,7 @@ public ToolTipData(string title, string text)
         {
             if (string.IsNullOrEmpty(title))
             {
-                throw new ArgumentException("title cannot be null or empty", nameof(title));
+                throw new ArgumentException(Resource.ToolTipData_TitleEmptyError, nameof(title));
```

Comment:
Can this happen on a user input or it can only happen if there is a coding error?

Reply:
After brief look, I would say it can't happen on a user input, but as a internal search result error.

Reply:
Does it get logged or is it shown to the user?
In the first case, add a comment saying the string should not be localized.
If the messages is shown to the user, I see very little value in localizing and showing it to the user, we should simply show  an error code (we don't want to do reverse lookup of translated error messages).